modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/DXPActivationKeysTable/utils/getTooltipTitles.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+export function getTooltipTitles({title}) {
+	const activationNames = title.split(',');
+
+	if (activationNames.length) {
+		return (
+			<div>
+				<p className="font-weight-bold m-0">{activationNames[0]}</p>
+
+				<p className="font-weight-normal m-0 text-paragraph-sm">
+					{activationNames[1]}
+				</p>
+			</div>
+		);
+	}
+
+	return activationNames[0];
```

Comment:
If activationNames.length is zero the last return will broken? 

```suggestion
	const activationNames = title?.split(',');

	if (activationNames?.length) {
		return (
			<div>
				<p className="font-weight-bold m-0">{activationNames[0]}</p>

				<p className="font-weight-normal m-0 text-paragraph-sm">
					{activationNames[1]}
				</p>
			</div>
		);
	}

	return;
```